windows/scripts/VMware-tools/remove-tools.ps1

```diff
@@ -0,0 +1,34 @@
+
+Write-Host "`n`nRemoving VMware tools from your VM machine`n`n"   -ForegroundColor red -BackgroundColor white
+
+function removevmtools {
+
+try {
+$product = Get-WmiObject -Class Win32_Product | Where-Object { $_.Name -like "VMware Tools"}
+$AppGUID = $product.properties["IdentifyingNumber"].value.toString()
+if ( $AppGUID.count -eq 0 )
+{
+
+  Write-Host  "`n`nVMware Tools not found`n`n"   -ForegroundColor red -BackgroundColor white
+  Write-Host "`n`nPlease Shtudown The VM and Export it as OVA`n`n"   -ForegroundColor red -BackgroundColor white
+
+}
+
+else {
+
+MsiExec.exe /norestart /q/x $AppGUID REMOVE=ALL
+Start-Sleep -s 15
+Write-Host  "`n`nVMware Tools Has Been Uninstalled`n`n"   -ForegroundColor red -BackgroundColor white
+Write-Host "`n`nPlease Shtudown The VM and Export it as OVA`n`n"   -ForegroundColor red -BackgroundColor white
+}
+
+
+}
+catch{
+ Write-Host  "`n`nVMware Tools not found`n`n"   -ForegroundColor red -BackgroundColor white
```

Comment:
These are assumptions. Anything could be going wrong. Just stating that it has been uninstalled is not correct and misleading. Its better to not catch anything so that the user gets a proper error & status unless you know what error to trap and how to handle it.